Gordon360/Authorization/StateYourBusiness.cs

```diff
@@ -188,6 +188,8 @@ private async Task<bool> CanReadOneAsync(string resource)
                 }
             case Resource.NEWS:
                 return true;
+            case Resource.STUDENT_SCHEDULE:
+                return user_groups.Contains(AuthGroup.Advisors);
```

Comment:
This won't let people view their own schedules, will it?

Also, the current rule is that anyone can view a faculty member's course schedule.

Please update the authorization to reflect these rules.